server/chat.ts

```diff
@@ -1701,6 +1701,27 @@ export const Chat = new class {
 		htmlContent = htmlContent.replace(/\n/g, '&#10;');
 		return htmlContent;
 	}
+	getReadmoreCodeBlock(str: string) {
+		const params = str.substr(+str.startsWith('\n')).split('\n');
+		if (params.length === 1 && params[0].length < 80 && !params[0].includes('```')) {
+			return `<code>${params[0]}</code>`;
+		}
```

Comment:
This should be moved back out; it's a special case of `!code` that's not really relevant to other use-cases.

```suggestion
```